packages/rocketchat-sharedsecret/sharedsecret.js

```diff
@@ -0,0 +1,87 @@
+const SharedSecret = [];
+
+if (Meteor.isServer) {
+	const EncryptMessage = ((message) => {
```

Comment:
Lets use named function for theses cases (high level functions / were we used classes before) to help the stack trace.

`function EncryptMessage(message) {`